pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "alfalfa"
+version = "0.1.0"
```

Comment:
ok. maybe this is the version of record. version of record for computers? changelog being the human facing version.

Reply:
Yes, the docker publishing script currently parses this.